Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# fork of the domain_name gem
+gem 'domain_name', '= 0.5.25', path: 'vendor/gems/domain_name-0.5.25'
```

Comment:
Does this actually work for the `.gemspec` too, when you install it using `gem install`?